index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Todoリスト</title>
+  <link rel="stylesheet" href="index.css">
+  <script src="https://unpkg.com/vue@2.7.11/dist/vue.min.js"></script>
+</head>
+<body>
+
+  <div id="app">
+    <input type="text" v-model="addText" placeholder="Todoを入力してください">
+    <button v-on:click="addTodo()">追加</button>
+    <ul>
+      <li v-for="todo in allTodos" :key="todo.id">
+        <div v-if="todo.edit===false">
```

Comment:
```suggestion
        <div v-if="!todo.edit">
```

booleanの値をifで参照するときは`=== false`や`=== true`は書かないことが多いです